static/app/utils/replays/hooks/useCrumbHandlers.tsx

```diff
@@ -1,10 +1,11 @@
-import {useCallback} from 'react';
+import {useCallback, useMemo} from 'react';
+import debounce from 'lodash/debounce';
 
 import {useReplayContext} from 'sentry/components/replays/replayContext';
 import {relativeTimeInMs} from 'sentry/components/replays/utils';
 import {Crumb} from 'sentry/types/breadcrumbs';
 
-function useCrumbHandlers(startTimestampMs: number = 0) {
+function useCrumbHandlers(startTimestampMs: number = 0, hoverDebounce: boolean = true) {
```

Comment:
Is there a case where we wouldn't want to debounce?

Reply:
I was thinking of the case where a breadcrumb might not be part of a list. IE: hovering over an item in the timeline could be one to a few items and wouldn't really benefit from the debounce. But it probably wouldn't be that big of a deal anyways just figured it would be useful. LMK what you think I can pull that part out for sure